.github/workflows/_qa.yaml

```diff
@@ -24,6 +24,7 @@ jobs:
 
       - run: npm install --ci
       - run: npx eslint --max-warnings=0
+      - run: npx tsc -p .
       - run: npx tsc -p e2e
```

Comment:
is the first line automatically including the second, more narrow?

Reply:
The project at the root excludes the e2e folder

https://github.com/Aam-Digital/ndb-core/blob/8def237856a75a92bbd7bb6b6999190748858c0e/tsconfig.json#L52-L54

This is necessary (I think) for the app code to build properly. But we still want to type-check the e2e test code and this is what the second line is for.

Reply:
Ah, thanks for the explanation :+1: